### ISSUE TITLE
Improve visualisation sections for the basics and for Plotly reporting

### ISSUE THREAD

**Issue by stichbury** [kedro-org/kedro#1949]
## Description

The Kedro-Viz section of the docs needs to be updated. There are currently 3 pages and this issue is to address the first 2 pages "Get started" and "Visualise charts" in detail to make improvements for readability, clarity and correctness.

The final section on experiment tracking should have a cosmetic set of fixes to remove mention of earlier sections of the tutorial (since this is no longer part of that spaceflights tutorial) but I'll leave a full update to a separate issue.

**Comment by stichbury** [comment]
There's a PR outstanding for this (see above #2061) but we are blocked on a bug in the example code (explained on Slack https://github.com/kedro-org/kedro-viz/issues/1205) and a bug in Kedro Viz (https://github.com/kedro-org/kedro-viz/issues/1205)
